frontend/src/lib/constants.tsx

```diff
@@ -164,3 +165,8 @@ export const SSOProviderNames: Record<SSOProviders, string> = {
     gitlab: 'GitLab',
     saml: 'Single sign-on (SAML)',
 }
+
+// TODO: Support checking minimum plan required for specific feature and highlight the relevant plan in the
+// pricing page (or billing page). Requires updating the pricing page to support this highlighting first.
+export const UPGRADE_LINK = (cloud?: boolean): { url: string; target?: '_blank' } =>
+    cloud ? { url: urls.organizationBilling() } : { url: 'https://posthog.com/pricing', target: '_blank' }
```

Comment:
This feels like something that will expand and be refactored soon enough... but fine to start somewhere.

Reply:
Probably we'll want to do something with better UX but as you said, good start.